server/src/controllers/employer/getAllEmployers.js

```diff
@@ -23,7 +23,9 @@ const getAllEmployersRequestHandler = async (req, res) => {
       query.name = { [Op.like]: `%${employerName}%` };
     }
     if (phoneNumber) {
-      query.phone_number = { [Op.like]: `%${phoneNumber}%` };
+      query.phone_number = Sequelize.literal(
+        `REGEXP_REPLACE(phone_number, '[^0-9]', '') REGEXP '${phoneNumber}'`,
+      );
```

Comment:
I'm not 100% sure but I doubt this query will actually sanitize the phoneNumber since we used sequelize.literal